pandas/tests/frame/test_to_csv.py

```diff
@@ -764,7 +765,7 @@ def test_to_csv_wide_frame_formatting(self):
         # Issue #8621
         df = DataFrame(np.random.randn(1, 100010), columns=None, index=None)
         with ensure_clean() as filename:
-            df.to_csv(filename, header=False, index=False)
+            df.to_csv(filename, header=False, index=False, index_label=False)
```

Comment:
the default for `index_label` is `None`, but it probably makes sense for the default to _be_ `False` when `header=False`. This would maintain backwards compatibility and follow the Principle of Least Astonishment.

Reply:
you are right!!! thanks for the review and feedback! @simonjayhawkins just changed!

Reply:
you will also need to change the docstring. to make it less wordy. it may be worth allowing `True` as an option, having `True` _as_ the default when `header` and `index` are `True` and then the current default `None` will just become the new default. i.e. True use the column and index names if they exist and so maintain backwards compatibility.

Reply:
thanks for the feedback, I am not sure if this is worth it... since when `header` and `index` are True, then current default `None` plays exactly same role as `True`, which will print index name is it exists, so the users don't have to define it. And I haven't found other use cases for `index_label` being `True`. Please enlighten me if you have different thoughts. @simonjayhawkins